component/card/card.css

```diff
@@ -0,0 +1,176 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+@import url("../button/button.css");
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  overflow-x: hidden;
+}
+* {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.card h2,
+h4,
+h5 {
+  margin: 0;
+}
+.responsive-img {
+  max-width: 100%;
+}
```

Comment:
I think these styles are common for all , if it is you can move to `main css file`. And keep only css required to `card component` in this file.

Reply:
already added forget to remove done